pom.xml

```diff
@@ -1,149 +1,234 @@
+							<mainClass>${assemblyMainClass}</mainClass>
+						</manifest>
+					</archive>
+					<descriptorRefs>
+						<descriptorRef>jar-with-dependencies</descriptorRef>
+					</descriptorRefs>
+				</configuration>
+				<executions>
+					<execution>
+						<id>make-assembly</id> <!-- this is used for inheritance merges -->
+						<phase>package</phase> <!-- bind to the packaging phase -->
+						<goals>
+							<goal>single</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<!-- Launch4 J plugin to create cross plattform java app -->
+			<plugin>
+				<groupId>com.akathist.maven.plugins.launch4j</groupId>
+				<artifactId>launch4j-maven-plugin</artifactId>
+				<version>1.7.22</version>
+				<executions>
+					<execution>
+						<id>l4j-gui</id>
+						<phase>package</phase>
+						<goals>
+							<goal>launch4j</goal>
+						</goals>
+						<configuration>
+							<dontWrapJar>false</dontWrapJar>
+							<headerType>console</headerType>
+							<jar>target/jmeld-${project.version}-jar-with-dependencies.jar</jar>
+							<outfile>${project.build.directory}/jmeld.exe</outfile>
+							<errTitle />
+							<cmdLine />
+							<chdir />
+							<priority>normal</priority>
+							<downloadUrl>http://java.com/download</downloadUrl>
+							<supportUrl>https://github.com/albfan/jmeld/issues</supportUrl>
+							<customProcName>false</customProcName>
+							<stayAlive>false</stayAlive>
+							<manifest />
+							<icon />
+							<jre>
+								<path />
+								<minVersion>1.8.0</minVersion>
+								<maxVersion />
+								<jdkPreference>preferJre</jdkPreference>
+								<initialHeapSize>256</initialHeapSize>
+								<maxHeapSize>3000</maxHeapSize>
+							</jre>
+							<!-- <splash> <file>${project.basedir}/src/main/build/Sunflower.bmp</file> 
+								<waitForWindow>true</waitForWindow> <timeout>60</timeout> <timeoutErr>true</timeoutErr> 
+								</splash> -->
+							<versionInfo>
+								<fileVersion>0.0.${project.version}</fileVersion>
+								<txtFileVersion>${project.version}</txtFileVersion>
+								<fileDescription>JMeld</fileDescription>
+								<copyright>albfan</copyright>
+								<productVersion>0.0.1.0</productVersion>
+								<txtProductVersion>${project.version}</txtProductVersion>
+								<productName>JMeld</productName>
+								<companyName>albfan</companyName>
+								<internalName>jmeld</internalName>
+								<originalFilename>jmeld.exe</originalFilename>
+							</versionInfo>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<!-- http://kosalads.blogspot.de/2013/04/maven-copy-artifact-in-to-separate.html 
+				https://stackoverflow.com/a/16862404/1497139 -->
+			<plugin>
+				<artifactId>maven-antrun-plugin</artifactId>
+				<configuration>
+					<tasks>
+						<copy
+							file="target/jmeld-${project.version}-jar-with-dependencies.jar"
+							tofile="release/jmeld.jar" />
+						<copy file="target/jmeld.exe" tofile="release/jmeld.exe" />
```

Comment:
This only solves fat jar for windows (exe files) Ideally it should create a executable jar. For me is ok to generate specific OS releases but an agnostic one is the first goal